### ISSUE TITLE
Feature request: users name rather than id

### ISSUE THREAD

**Issue by karye** [github/gh-classroom#29]
When cloning student’s assignment repository the naming of the folders is "assignementname-userid".
It would be most useful to have "assignementname-username" since the userid:s can be quite cryptic.
Resulting in 30 folder-repos you wonder whose their are.

**Comment by shanep** [comment]
Hi @karye yes this would be nice, I want this feature as well. I when to implement it and ran into the fact that not all students actually have their name set in their GitHub profile. So in those cases we would have to fall back to their GitHub username which would be confusing for teachers and TA's. 

The real solution is to have the roster identifier included in the Student struct (issue #9 ) and ideally add the roster to the backend REST API (issue #36) 

Unfortunately both issues (#9, #36) required changes to the back end API which is not open source so I can't implement them myself. If those to issues are resolved I can push these changes as well :)

**Comment by karye** [comment]
Yes, the student rooster-name should be used when creating student-repos.

**Comment by pmcstjean** [comment]
+1 for the student roster name ...I made myself a python script that update all the folders and rename them to the roster name (by loading the csv file first). Problem is, I can't use the new classroom "pull" feature because it doesn't recognize the folders.
